Pyrado/pyrado/algorithms/utils.py

```diff
@@ -277,3 +277,6 @@ def __getstate__(self):
     def __setstate__(self, state):
         """Required so that pickle does not end in an endless recursion"""
         vars(self).update(state)
+
+
+SamplerBase.register(RolloutSavingWrapper)
```

Comment:
This registers RolloutSavingWrapper as a virtual subclass of samplerbase. An instance `r` of a RolloutSavingWrapper then passes any `isinstance(r, SamplerBase)` checks.

Reply:
I see, it has (for the type checking) the same effect as making `RolloutSavingWrapper` a subclass of `SamplerBase`. We could actually do make it a subclass and then forward all the required/abstract methods to the wrapped sampler. this would be in line with the way the env wrappers are made.

Reply:
But please write an expressive doc string on this, since one cant not expect that others will understand this without comments.